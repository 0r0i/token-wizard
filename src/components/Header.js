import React from 'react'
import '../assets/stylesheets/application.css';
import { Link } from 'react-router-dom'

export const Header = () => (
	<header className="header">
    <div className="container">
      <Link to='/'><a href="#" className="logo"></a></Link>
    </div>
  </header>
)

