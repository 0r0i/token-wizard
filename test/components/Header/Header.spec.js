import React from 'react'
import { Header } from '../../../src/components/Header/index'
import Adapter from 'enzyme-adapter-react-15'
import { configure, mount, shallow } from 'enzyme'
const displayHeaderAndFooterInIframe = require('../../../src/utils/utils').displayHeaderAndFooterInIframe
const mockTrue = { displayHeaderAndFooterInIframe: true }
const mockFalse = { displayHeaderAndFooterInIframe: false }

configure({ adapter: new Adapter() })

describe('Header ', () => {

  it(`should render Footer component`, () => {
    jest.mock('../../../src/utils/utils', () => mockTrue)
    const wrapper = shallow(
      <Header

      />
    )
    expect(wrapper).toMatchSnapshot()
  })

  it(`should return null`, () => {
    jest.mock('../../../src/utils/utils', () => mockFalse)

    const wrapper = shallow(
      <Header

      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
