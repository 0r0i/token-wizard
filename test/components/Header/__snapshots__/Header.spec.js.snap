// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header  should render Footer component 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <Link
      className="logo"
      replace={false}
      to="/"
    />
  </div>
</header>
`;

exports[`Header  should return null 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <Link
      className="logo"
      replace={false}
      to="/"
    />
  </div>
</header>
`;
